Docs/source/running_cpp/parameters.rst

```diff
@@ -270,6 +270,12 @@ Particle initialization
     * ``NRandomPerCell``: injection with a fixed number of randomly-distributed particles per cell.
       This requires the additional parameter ``<species_name>.num_particles_per_cell``.
 
+    * ``singleparticle``: Inject a single macroparticle.
+      This requires the additional parameters:
+      ``<species_name>.single_particle_pos`` (`3 doubles`, particle 3D position)
```

Comment:
Thanks for giving the dimensions/definitions of parameters below! Could you also add this for the position? e.g.,
```suggestion
      ``<species_name>.single_particle_pos`` (`3 doubles`, particle 3D position [meters])
```

Reply:
This was a great catch @MaxThevenet ! Would you like me to add units to the definition of the `gaussian_beam` injection style (lines below) too?